integreatly/integreatly-1.9.0/integreatly-operator.1.9.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,390 @@
+        version: v1alpha1
+        kind: Installation
+        displayName: Integreatly Installation
+        description: Represents an installation of Integreatly.
+        specDescriptors:
+          - description: The desired integreatly products.
+            displayName: Products
+            path: products
+  install:
+    strategy: deployment
+    spec:
+      clusterPermissions:
+        - serviceAccountName: integreatly-operator
+          rules:
+            ## start backup permissions
+            - apiGroups:
+                - batch
+              resources:
+                - cronjobs
+              verbs:
+                - create
+                - update
+            - apiGroups:
+                - "rbac.authorization.k8s.io"
+              resources:
+                - "clusterroles"
+                - "clusterrolebindings"
+              verbs:
+                - create
+                - update
+            - apiGroups:
+                - "*"
+              resources:
+                - "*"
+              verbs:
+                - get
+                - list
+            - apiGroups:
+                - ""
+              resources:
+                - "pods/exec"
+              verbs:
+                - create
+                - update
+              ## end backup permissions
+            - apiGroups:
+                - "integreatly.org"
+              resources:
+                - "*"
+              verbs:
+                - "*"
+            # This is used to allow the operator to setup the ns needed for each product
+            - apiGroups:
+                - ""
+              resources:
+                - namespaces
+                - namespaces/finalizers
+                - "serviceaccounts"
+              verbs:
+                - list
+                - create
+                - watch
+                - get
+                - update
+            # This is used to allow the operator to setup catalog source config and remove it if needed.
+            - apiGroups:
+                - operators.coreos.com
+              resources:
+                - catalogsourceconfigs
+              verbs:
+                - list
+                - create
+                - watch
+                - delete
+            - apiGroups:
+                - ""
+              resources:
+                - pods
+                - secrets
+              verbs:
+                - list
+                - get
+            - apiGroups:
+                - operators.coreos.com
+              resources:
+                - catalogsourceconfigs/finalizers
+              verbs:
+                - '*'
+            - apiGroups:
+                - operators.coreos.com
+              resources:
+                - operatorgroups
+                - subscriptions
+                - installplans
+              verbs:
+                - list
+                - create
+                - watch
+                - get
+                - delete
+                - update
```

Comment:
Adding `update` permission here.
All other permissions are supposed to stay the same as in the previous version.